src/main/scala/ai/privado/exporter/SinkExporter.scala

```diff
@@ -55,8 +46,66 @@ class SinkExporter(cpg: Cpg) {
     convertSinkList(sinkTagList)
   }
 
-  def getProcessing: List[SinkProcessingModel] = {
+  def getProbableSinks: List[String] = {
+    var blackListedGroupIds: List[String] = List(
```

Comment:
Convert this skiplist into config rule files picked from privado skiplist.yml configuration kept inside `config` folder.